src/status_im/contexts/settings/wallet/saved_addresses/add_address_to_save/view.cljs

```diff
@@ -163,9 +163,8 @@
                                                (rf/dispatch
                                                 [:open-modal :screen/settings.save-address]))
                                              [address ens-name? address-or-ens])]
-    (rn/use-mount (fn []
-                    (rf/dispatch [:wallet/clean-scanned-address])
-                    (rf/dispatch [:wallet/clear-address-to-save])))
+    (rn/use-unmount #(rf/dispatch [:wallet/clean-scanned-address]))
```

Comment:
Moved to unmount as we already clean the scanned address on navigating to scanner screen in saved-address